packages/now-node/src/dev-server.ts

```diff
@@ -21,22 +21,22 @@ register({
   transpileOnly: true,
 });
 
-import http from 'http';
-import path from 'path';
+import { createServer, Server, IncomingMessage, ServerResponse } from 'http';
+import { join } from 'path';
+import { Readable } from 'stream';
+import { Bridge } from './bridge';
 import { createServerWithHelpers } from './helpers';
 
-function listen(
-  server: http.Server,
-  port: number,
-  host: string
-): Promise<void> {
+function listen(server: Server, port: number, host: string): Promise<void> {
   return new Promise(resolve => {
     server.listen(port, host, () => {
       resolve();
     });
   });
 }
 
+let bridge = new Bridge();
+
 async function main() {
```

Comment:
Should we utilize the `makeNowLauncher()` function since these seems almost identical?

Reply:
And like `eval()` it?

Reply:
Maybe. Either `eval()` the string or change the code to be a proper function and `fun.toString()`?

Another option is to write the string to a file like we do for production deployments.

I'm just trying to keep this DRY because someone is going to fix something in prod and forget to update dev.

Reply:
Maybe we can enforce running the same test suite against dev and cloud? So that adding a feature to cloud would require you to add it to dev.

Reply:
@lucleray That basically is already the case with the "Dev" integration tests in CLI. We deploy the code to prod and compare the status codes, headers, etc. with the `vc dev` version.

Reply:
The problem is that builders/runtimes are not executed in dev until they're published, right?

Reply:
This PR has been reformatted to make the launcher logic be actual functions, which are invoked by the dev server directly, and `.toString()`'d in production to reuse even more logic.